### PR TITLE
Column

### LINKED ISSUE #4: mj-column

### PR DESCRIPTION
**Pull request by LiamRiddell**
Closes #4

### CODE REVIEW

File: Mjml.Net/Components/Body/ColumnComponent.cs [hunk 1]
```diff
@@ -1,11 +1,288 @@
...
+        public ContainerWidth ContainerWidth;
+        public double CurrentWidth;
 
         public override void Render(IHtmlRenderer renderer, GlobalContext context)
         {
+            ContainerWidth = context.GetContainerWidth();
+
+            var (width, widthString, pixels) = GetParsedWidth();
+
+            var classesName = $"{GetColumnClass(width, widthString, context)} mj-outlook-group-fix";
```
Comment (SebastianStehle) [inline Mjml.Net/Components/Body/ColumnComponent.cs:87]: Please just use the class method in the renderer for that. See mj-group.

File: Tests/ColumnTests.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+            AssertHelpers.HtmlAssert(TestHelper.GetContent("ColumnTwo.html"), result);
+        }
+
+        [Fact]
+        public void Should_render_column_three()
+        {
+            var source = @"
+<mjml-test head=""false"">
+<mj-column background-color=""red""></mj-column>
```
Comment (SebastianStehle) [inline Tests/ColumnTests.cs:36]: Can you indent that like in other tests?

**Review by SebastianStehle — CHANGES_REQUESTED** [review]

**Comment by SebastianStehle** [comment]
The main branch builds fine, why not this one?

**Comment by SebastianStehle** [comment]
LGTM. I don't what the problem with the build is, but you can Squash merge it.

**Comment by LiamRiddell** [comment]
> The main branch builds fine, why not this one?

I think I just needed to update the branch from main again.

**Pull request merged**